src/components/pages/GamePlay/GamePlayMain.js

```diff
@@ -68,7 +69,7 @@ function GamePlayMain(props) {
         <Route component={NotFoundPage} />
       </Switch>
 
-      <ChildFooter />
+      {pathname != '/gameplay/mission/read' ? <ChildFooter /> : null}
```

Comment:
This route is obsolete.  Please DM Tony Miller and he will give you the correct route path.  A major route refactor just completed, so please make sure the team is oriented to the correct routes.  Tony will await your message.  Thank you.

Reply:
I found it odd, too, and asked about it but got no response. Do we happen to know where any of this was documented so that I can review the status?

Reply:
Who did you reach out to regarding this? Did you console.log check pathname to make sure the props received are the props you're looking for? If we want the ChildFooter to render only for one page but not the others do we want the child footer on the page at all?